src/lib/pickles/step_verifier.ml

```diff
@@ -488,17 +508,34 @@ struct
                         | `Packed_bits (b, n) ->
                             (b, n) ) ) )
         in
+
+        (* absorb x_hat *)
         absorb sponge PC x_hat ;
+
+        (* absorb w_comm *)
         let w_comm = messages.w_comm in
         Vector.iter ~f:absorb_g w_comm ;
+
+        (* sample beta *)
```

Comment:
Do you think the comment `sample beta` is helpful in a way the code `let beta = sample ()` is not?

Reply:
I guess it's a matter of taste but I'm:

- trying to make it symmetrical. If I'm commenting on each step of the protocol, then why would I omit this step?
- trying to make it useful. If you're scanning for a step of the protocol, browsing through each comment, then you'll be able to find this one

Reply:
I would tend to agree with Izaak that these comments add nothing. If anything, they probably have negative value, since there's more to update if things change, and it inflates the size of the function with redundant information.
Can we remove them and stick to just the doc-comment?